ui/apps/platform/src/Containers/MainPage/Notifications.tsx

```diff
@@ -8,6 +8,11 @@ function Notifications(): ReactElement {
     const notifications = useSelector(selectors.notificationsSelector);
 
     return (
+        /*
+        (dv 2024-05-01) 
+        Upgrading to React types 18 causes a type error below due to the `children` prop being removed from the `React.FC` type
+
+        @ts-expect-error ToastContainer does not expect children as a prop */
```

Comment:
There are a handful of places where a 3rd party dependency is typed as `React.FC<SomeCustomProps>`, but React 18 removes the automatic addition of `children` to the type of props passed to `React.FC`.

Long term we will want to update these dependencies if and when they fix their types, but this fixes our build errors short term.